app/src/main/java/org/oppia/android/app/devoptions/forcenetworktype/ForceNetworkTypeViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.android.app.devoptions.forcenetworktype
+
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import org.oppia.android.util.networking.NetworkConnectionUtil
+import javax.inject.Inject
+
+/**
+ * [ViewModel] for [ForceNetworkTypeFragment]. It populates the recyclerview with a list of
```

Comment:
I suggest either

```suggestion
 * [ViewModel] for [ForceNetworkTypeFragment]. It populates the RecyclerView with a list of
```

or

```suggestion
 * [ViewModel] for [ForceNetworkTypeFragment]. It populates the recycler view with a list of
```

("recyclerview" isn't a word, but "RecyclerView" is a proper noun versus "recycler view" which spells it out).

Ditto everywhere else.

Reply:
Done. Followed the below one. Thanks.